packages/backend/src/Controllers/Ballot/getBallotsByElectionIDController.ts

```diff
@@ -20,8 +20,20 @@ const getBallotsByElectionID = async (req: IElectionRequest, res: Response, next
         Logger.info(req, msg);
         throw new BadRequest(msg)
     }
-    Logger.debug(req, "ballots = ", ballots);
-    res.json({ election: req.election, ballots: ballots })
+
+    // Scrub identifying information from ballots to preserve voter anonymity
+    const scrubbedBallots = ballots.map(ballot => ({
```

Comment:
This looks reasonable to me. This API is only used for the view ballot page and that page only cares about the ballot anyway